constants.js

```diff
@@ -68,6 +68,8 @@ const constants = {
     maximumAllowedPartSize: process.env.MPU_TESTING === 'yes' ? 104857600 :
         5368709120,
 
+    maximumMetaHeadersSize: 2136,
```

Comment:
please add note referencing aws documentation and then comment about what they actually allow